webapp/portlet/src/main/webapp/vue-apps/space-top-bannerlogo/components/ExoSpaceLogoBanner.vue

```diff
@@ -122,7 +123,9 @@
               <exo-space-favorite-action
                 v-if="favoriteActionEnabled"
                 :is-favorite="isFavorite"
-                :space-id="spaceId" />
+                :space-id="spaceId"
+                @added="popoverActionEvent('bookmarkAction')"
+                @removed="popoverActionEvent('unBookmarkAction')" />
```

Comment:
IMO, you should use the events favorite-added and favorite-removed that already defined without adding new operations, we already use the **Bookmark** and **UnBookmark** operations